internal/controllers/machinehealthcheck/machinehealthcheck_targets.go

```diff
@@ -93,6 +94,12 @@ func (t *healthCheckTarget) needsRemediation(logger logr.Logger, timeoutForMachi
 	var nextCheckTimes []time.Duration
 	now := time.Now()
 
+	if annotations.HasRemediateMachine(t.Machine) {
+		conditions.MarkFalse(t.Machine, clusterv1.MachineHealthCheckSucceededCondition, clusterv1.RemediateMachineReason, clusterv1.ConditionSeverityWarning, "Marked for manual remediation")
```

Comment:
```suggestion
		conditions.MarkFalse(t.Machine, clusterv1.MachineHealthCheckSucceededCondition, clusterv1.RemediateMachineReason, clusterv1.ConditionSeverityWarning, "Marked for remediation via remediate-machine annotation")
```
(similar in log below)